source/production/F0.Minesweeper.Components/Pages/Game/Game.razor.cs

```diff
@@ -18,11 +18,13 @@ public partial class Game : IDisposable
 		protected override void OnParametersSet()
 		{
 			EventAggregator?.GetEvent<DifficultyLevelChangedEvent>().Subscribe(OnDifficultyChanged);
+			EventAggregator?.GetEvent<RestartGameEvent>().Subscribe(RestartGame);
 		}
 
 		void IDisposable.Dispose()
 		{
 			EventAggregator?.GetEvent<DifficultyLevelChangedEvent>().Unsubscribe(OnDifficultyChanged);
+			EventAggregator?.GetEvent<RestartGameEvent>().Unsubscribe(RestartGame);
```

Comment:
Should we unsubscribe in reverse order of subscribing? Well ... is this relevant?

Reply:
Since it's an unsubscription I'd say that's hardly relevant. My reasoning is that the events have no dependencies on each other and that's why I do not see any issues with the way it is right now.
I also couldn't find any best practices from Prism in that regard. Is it ok to leave this as it is for you too?

Reply:
It definitely is. Thanks.